ivy/array/activations.py

```diff
@@ -60,10 +60,10 @@ def gelu(
 
         Examples
         --------
-        >>> x = ivy.array([0.3, -0.1])
-        >>> y = x.gelu()
+        >>> x = ivy.array([-1.2, -0.6, 1.5])
+        >>> y = ivy.gelu(x)
```

Comment:
Shouldn't this be `x.gelu()`?

Reply:
Yea, You are right, going to change it now.